### PR TITLE
Change Detune To Pitch Bend in Piano Roll For Stable-1.2

### LINKED ISSUE #4174: Rename the "detune" button in Piano Editor as "pitch-bend" or "portamento"
When we say "detuning" of a real-world musical instrument, we really mean changing of its default scale. (=scale-shifting)

For example, changing the tension of strings in stringed instruments or certain drums (e.g. Tabla, pakhavaj, mrudangam).

This changes the playing characteristic of the instrument. 

But in  Piano Editor, the button that is called "detune" actually adds a "pitch-bend" or "portamento" effect. (usually done with a pitch-wheel in a synth).

It does _not_ change the pitch of the instrument (no shift of scale).

Therefore it should be renamed as "pitch-bend" or "portamento".
(The former name is pop form and the latter name is classical/formal form.)

### PR DESCRIPTION
**Pull request by Anonymouqs**
Fixes #4174 in order to give a clearer name to the mode in the Piano Roll.

This Pull Request is for the 1.2 Stable Branch while there is another Pull Request for the Master Branch.

### CODE REVIEW

File: src/gui/editors/PianoRoll.cpp [hunk 1]
```diff
@@ -4077,14 +4077,14 @@ PianoRollWindow::PianoRollWindow() :
 	QAction* drawAction = editModeGroup->addAction( embed::getIconPixmap( "edit_draw" ), tr( "Draw mode (Shift+D)" ) );
 	QAction* eraseAction = editModeGroup->addAction( embed::getIconPixmap( "edit_erase" ), tr("Erase mode (Shift+E)" ) );
 	QAction* selectAction = editModeGroup->addAction( embed::getIconPixmap( "edit_select" ), tr( "Select mode (Shift+S)" ) );
-	QAction* detuneAction = editModeGroup->addAction( embed::getIconPixmap( "automation" ), tr("Detune mode (Shift+T)" ) );
+	QAction* pitchBendAction = editModeGroup->addAction( embed::getIconPixmap( "automation" ), tr("Detune mode (Shift+T)" ) );
```
Comment (tresf) [inline src/gui/editors/PianoRoll.cpp]: Did you mean to change the text for this as well?

Comment (Anonymouqs) [reply src/gui/editors/PianoRoll.cpp]: Yeah, so the source code is clearer.

Comment (tresf) [reply src/gui/editors/PianoRoll.cpp]: @Anonymouqs I do not understand your reply.  Did you make a mistake and forget this line?  Will you fix it? (Hint, the text still reads `"Detune mode (Shift+T)"`

**Comment by Anonymouqs** [comment]
Added the fix. Apologies for the confusion.

File: src/gui/editors/PianoRoll.cpp [hunk 2]
```diff
@@ -4077,14 +4077,14 @@ PianoRollWindow::PianoRollWindow() :
 	QAction* drawAction = editModeGroup->addAction( embed::getIconPixmap( "edit_draw" ), tr( "Draw mode (Shift+D)" ) );
 	QAction* eraseAction = editModeGroup->addAction( embed::getIconPixmap( "edit_erase" ), tr("Erase mode (Shift+E)" ) );
 	QAction* selectAction = editModeGroup->addAction( embed::getIconPixmap( "edit_select" ), tr( "Select mode (Shift+S)" ) );
-	QAction* detuneAction = editModeGroup->addAction( embed::getIconPixmap( "automation" ), tr("Detune mode (Shift+T)" ) );
+	QAction* pitchBendAction = editModeGroup->addAction( embed::getIconPixmap( "automation" ), tr("Pitch Bend Mode (Shift+T)" ) );
```
Comment (DomClark) [inline src/gui/editors/PianoRoll.cpp]: "Mode" here ought to start with a lowercase "m" for consistency with the other tooltips.

Comment (Anonymouqs) [reply src/gui/editors/PianoRoll.cpp]: Added a new commit.

**Comment by PhysSong** [comment]
@Anonymouqs You can find remaining `detune` or `detuning` by:
```bash
git grep -i detune -- include/ src/ plugins/ # for detune
git grep -i detuning -- include/ src/ plugins/ # for detuning
```
However, I think first three commits are sufficient because internal references are not important for stable releases. That may be done in `master` after synchronizing branches.

**Comment by Anonymouqs** [comment]
@PhysSong Should we discuss on #devtalk if we should revert the internal reference commits?

**Comment by PhysSong** [comment]
@Anonymouqs There have been some discussions on Discord `#devtalk`. Changing UI part and related variable names(which is done by first three commits) seems enough for `stable-1.2`. Further renaming can be done later(in `master`).

**Comment by Anonymouqs** [comment]
@PhysSong I just read your message when I just finished renaming all of the code, what should I do with the extra commits now? They all pass through the Travis Build.

**Comment by tresf** [comment]
Although I'm sympathetic to learning the PR process, 39 commits is growing excessive for a cosmetic change.  Let's please wrap this up.

**Pull request merged**